templates/getting_started/faq.md

```diff
@@ -493,51 +493,40 @@ model.fit(dataset, epochs=10, validation_data=val_dataset)
 
 ---
 
-### How can I regularly save Keras models during training?
+### How can I ensure my training with Keras can withstand program interruptions?
```

Comment:
Change the title to "How can I ensure my training run can recover from program interruptions?"

You will also need to update the link that points to this question (since the link follows the question content)

Reply:
Done.